packages/core/src/parser.ts

```
@@ -0,0 +1,302 @@
+  // Walk through the import clause to find specifiers
+  const traverse = (n: Parser.SyntaxNode) => {
+    switch (n.type) {
+      case 'identifier':
+        // Default import
+        if (n.parent?.type === 'import_clause') {
+          specifiers.push(n.text);
+        }
+        break;
+      case 'import_specifier':
+        // Named import: could be `foo` or `foo as bar`
+        const aliasNode = n.childForFieldName('alias');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)